src/app/components/ngx-tweet.component.ts

```diff
@@ -18,6 +18,7 @@ import { NgxTweetService } from '../services/ngx-tweet.service';
 })
 export class NgxTweetComponent implements OnInit {
     @Input() public tweetId: string;
+    @Input() public twitterConfig: any = {};
```

Comment:
It would be great if you could provide an Interface representing all available options.

Reply:
Hey
I could.. The reason I didn't was because of maintenance.. i.e trying to ensure it is always up-to-date with Twitter's implementation. My thought was that with a dirty "any" we just offload that work to the user, and anything invalid is just dropped.

Let me look into it later though!

Cheers

Jason

Reply:
Yeah I get that, but as it is Open Source, users are more than welcome to do just like what you did for the community: contribute and keep this lib as great as possible!

Thanks again.

Reply:
Roger that.. Will send something over later. Cheers